docs/ios/README.md

```diff
@@ -0,0 +1,31 @@
+# iOS distribution
+
+The source code can be found in [RNNativePlayground](https://github.com/kiwicom/mobile/tree/master/ios/RNNativePlayground)
+
+## Test playground app
+
+### KiwiHotels module
+
+1. To run `KiwiHotels` module with the packager, change [`RNKiwiConstants.m`](https://github.com/kiwicom/mobile/blob/master/ios/RNKiwiMobile/RNKiwiConstants.m) like follows:
```

Comment:
This step is not needed here. Probably we could create a separate section, called `Running from packager` or `Dev mode`.